### ISSUE TITLE
Disable zoom in Histogram

### ISSUE THREAD

**Issue by julianmlr** [oguzhaninan/vue-histogram-slider#29]
Hey, is it possible to disable the zoom via props so that the scale of the histogram remains fixed? 
![example](https://user-images.githubusercontent.com/26934306/101755463-2fd70d80-3ad5-11eb-81c6-59640245f6d4.jpeg)

**Comment by areindl** [comment]
Did you fix it somehow, @julianmlr ?

**Comment by julianmlr** [comment]
Not really. I just used the vuetify range slider. Of course this misses the histogram part, but I will just add the chart later by myself with chartjs: [Range Slider ](https://vuetifyjs.com/en/components/range-sliders/)

**Comment by hsks** [comment]
@areindl @julianmlr I briefly went through the source code. It looks like it does support a prop called "clip". When set to false (:clip="false"), it disables zoom and double click functionalities. This might come in handy in your cases.
